python/sglang/srt/layers/quantization/quark/quark_moe.py

```diff
@@ -0,0 +1,196 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
same pls be specific (to name and namespace), quark can be a lot more than mxfp4